src/pass/codegen/llvmgen.cpp

```diff
@@ -531,3 +531,33 @@ unique_ptr<llvm::Module> LLVMGen::Build(const Loop loop, llvm::LLVMContext& llct
     loop->Accept(llgen);
     return move(llgen._llmod);
 }
+
+void LLVMGen::register_vinstrs() {
+    const auto buffer = llvm::MemoryBuffer::getMemBuffer(llvm::StringRef(vinstr_str));
+
+    llvm::SMDiagnostic error;
+    std::unique_ptr<llvm::Module> vinstr_mod = llvm::parseIR(*buffer, error, llctx());
+    if (!vinstr_mod) {
+        throw std::runtime_error("Failed to parse vinstr bitcode");
+    }
+    if (llvm::verifyModule(*vinstr_mod)) {
+        throw std::runtime_error("Failed to verify vinstr module");
+    }
```

Comment:
Instead of manually checking and throwing runtime error like this, can we use llvm::cantFail() ?

Reply:
For both vinstr_mod and verifyModule